### PR TITLE
Initial OpenShift deployment information

### PR DESCRIPTION
**Pull request by isimluk**

### CODE REVIEW

File: docs/deployment/openshift/README.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+## Pre-requisites
+
+ - Have CrowdStrike CWP subscription with Falcon Container enabled (have your CrowdStrike CID ready)
+ - Have Container Administrator access to GCP and at least one GKE cluster deployed
+ - Create new CrowdStrike API key pair with permission to download the sensor (no other permission shall be required)
+
+## Installation Steps
+
+ - Authenticate your kubectl/oc with the OpenShift cluster
```
Comment (redhatrises) [inline docs/deployment/openshift/README.md:11]:
```suggestion
 - Authenticate to your OpenShift cluster
```

File: docs/deployment/openshift/README.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+   ```
+   oc login --token=sha256~abcde-ABCDE-1 --server=https://openshift.example.com
+   ```
+
+ - Create namespaces used by the operator
+   ```
+   kubectl create ns falcon-operator --dry-run=client -o yaml | kubectl apply -f
+   kubectl create ns falcon-system-configure --dry-run=client -o yaml | kubectl apply -f -
```
Comment (redhatrises) [inline docs/deployment/openshift/README.md:19]: For consistency, let's use `oc`

File: docs/deployment/openshift/README.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+   ```
+ 
+ - Add Falcon Operator subscription to the operator hub on the cluster (This is needed until falcon-operator is available through operatorhub.io)
+   ```
+   operator-sdk run bundle quay.io/crowdstrike/falcon-operator-bundle --namespace falcon-operator
+   ```
+
+ - Deploy FalconContainer either through OpenShift web console or through `kubectl`
+   ```
+   kubectl create -f https://raw.githubusercontent.com/CrowdStrike/falcon-operator/main/docs/deployment/openshift/falconcontainer-openshift.yaml --edit=true
```
Comment (redhatrises) [inline docs/deployment/openshift/README.md:29]: For consistency, let's use `oc`

File: docs/deployment/openshift/README.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+   ```
+   
+## Uninstall Steps
+
+ - To uninstall Falcon Container simply remove FalconContainer resource. The operator will uninstall Falcon Container product from the cluster.
+   ```
+   kubectl delete falconcontainers.falcon.crowdstrike.com default
```
Comment (redhatrises) [inline docs/deployment/openshift/README.md:36]: For consistency, let's use `oc`

File: docs/deployment/openshift/falconcontainer-openshift.yaml [hunk 5]
```diff
@@ -0,0 +1,14 @@
+﻿apiVersion: falcon.crowdstrike.com/v1alpha1
```
Comment (redhatrises) [inline docs/deployment/openshift/falconcontainer-openshift.yaml:1]: This feels like a duplicate file. Can't we just use what is in config?

**Review by redhatrises — CHANGES_REQUESTED** [review]

File: docs/deployment/openshift/falconcontainer-openshift.yaml (hunk 5 above)
Comment (isimluk) [reply docs/deployment/openshift/falconcontainer-openshift.yaml:1]: I could not find the file this is duplicate to?

File: docs/deployment/openshift/README.md (hunk 4 above)
Comment (isimluk) [reply docs/deployment/openshift/README.md:36]: 👍

File: docs/deployment/openshift/README.md (hunk 2 above)
Comment (isimluk) [reply docs/deployment/openshift/README.md:19]: 👍

**Pull request merged**